app/helpers/editorial_helper.rb

```diff
@@ -1,7 +1,9 @@
 module EditorialHelper
 
   def form_type(node_clazz)
-    "#{node_clazz.name}Form".constantize
+    # Because of development mode autoloading, the class may not exist until
+    # we call constantize. Hence just catching the exception.
+    "#{node_clazz.name}Form".constantize rescue NodeForm
```

Comment:
I'm not quite sure of the scenario in which this could hide real problems but it feels like it might do.